src/host/ConsoleArguments.cpp

```diff
@@ -22,6 +22,7 @@ const std::wstring_view ConsoleArguments::RESIZE_QUIRK = L"--resizeQuirk";
 const std::wstring_view ConsoleArguments::WIN32_INPUT_MODE = L"--win32input";
 const std::wstring_view ConsoleArguments::FEATURE_ARG = L"--feature";
 const std::wstring_view ConsoleArguments::FEATURE_PTY_ARG = L"pty";
+const std::wstring_view ConsoleArguments::COM_SERVER_ARG = L"-Embedding";

```

Comment:
this feels like a terrible nit, but all the other options are `--twoDashes` style, so this should probably be `--embedding`, right? 

(except `-ForceV1` which maybe raises more questions - maybe predates the other args)

Reply:
What super sucks here is that we don't have that choice. This exact argument with capital E and one - is just THE WAY that OutOfProc COM EXEs are started. (Unless there is a magic COM registry key that lets you specify something else...)